app/src/App.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <v-app id="app">
+    <v-toolbar
+      id="SecretBallot_Toolbar"
+      flat
+      app
+    >
+      <v-toolbar-title
+        id="SecretBallot_ToolbarText"
+        @click="$router.push('/')"
+      >
+        <img
+          id="SecretBallot_ToolbarIcon"
+          src="/assets/SecretBallot_icon.svg"
+        >
+        <b class="pl-2">Secret Ballot</b>
+      </v-toolbar-title>
+    </v-toolbar>
+
+    <v-content>
+      <div class="main">
+        <router-view/>
+      </div>
+    </v-content>
+
+    <v-footer
+      id="SecretBallot_Footer"
+      class="text-xs-center"
+      app
+    >
+      <v-container>
+        <v-flex xs12>
+          <v-card
+            id="SecretBallot_FooterCard"
+            align-center
+            flat
+          >
+            <v-card-text>
+              <div id="SecretBallot_FooterText">
+                This service is powered by the Oasis Network. <br />
```

Comment:
@armiller123 do the two lines of copy here look like they make sense?

Reply:
Is their a link or something that they're supposed to click on? Sorry trying to parse the code.

Reply:
It is supposed to link, but I haven't actually put in the links yet. Mostly just filler for now, until we finalize specific wording/what's most useful to link.